src/EventListener/AddToSearchIndexListener.php

```diff
@@ -27,6 +27,11 @@ class AddToSearchIndexListener
      */
     public function onKernelTerminate(PostResponseEvent $event)
     {
+        // FIXME: should be replaced with "Response implements Indexable" (see https://github.com/contao/symfony-todo/issues/3)
+        if (!defined('TL_ROOT')) {
```

Comment:
Why did you add an additional check? `Frontend::indexPageIfApplicable()` will decide itself when indexing is applicable (hence the method name).